docs/cudf/source/10min-cudf-cupy.ipynb

```diff
@@ -17,10 +17,8 @@
    "source": [
```

Comment:
<p>Ditto here, should we be recommending the fastest approach instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>